apps/builddao/widget/app.jsx

```diff
@@ -9,7 +9,7 @@ const config = {
     "--secondary-color": "white",
     background: "var(--main-color)",
     color: "var(--secondary-color)",
-    height: "100vh",
+    height: "100%",
```

Comment:
And this?

Reply:
The concern with this one is that now, in the preview you provided, the height of the page "grows" down as the page loads. With 100vh, the whole screen starts with full height instead